src/sentry/monitors/endpoints/organization_monitor_schedule_sample_data.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import cast
+
+from croniter import croniter
+from dateutil import rrule
+from django.utils import timezone
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.models.organization import Organization
+from sentry.monitors.schedule import SCHEDULE_INTERVAL_MAP
+from sentry.monitors.types import IntervalUnit
+
+
+@region_silo_endpoint
+class OrganizationMonitorScheduleSampleDataEndpoint(OrganizationEndpoint):
+    publish_status = {"GET": ApiPublishStatus.PRIVATE}
+    owner = ApiOwner.CRONS
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        num_ticks = int(request.GET.get("numTicks", 0))
+        schedule_type = request.GET.get("scheduleType")
+        cron_schedule = request.GET.get("cronSchedule")
+        interval = int(request.GET.get("intervalFrequency", 0))
+        unit = request.GET.get("intervalUnit")
+
+        if not num_ticks or num_ticks > 100 or not schedule_type:
```

Comment:
Let's make this a constant
```suggestion
        if not num_ticks or num_ticks > MAX_TICKS or not schedule_type:
```